RAW_RELEASE_NOTES.md

```diff
@@ -8,6 +8,11 @@ will make it substantially easier for the releaser to "linkify" all of the relea
 final version.
 
 ## 1.6.0
+* Added DOWNSTREAM_REMOTE_ADDRESS and DOWNSTREAM_REMOTE_ADDRESS_WITH_PORT access log formatters.
```

Comment:
I think this is backwards.  All the other address loggers (UPSTREAM_LOCAL_ADDRESS, DOWNSTREAM_LOCAL_ADDRESS) include ports.  I think DOWNSTREAM_REMOTE_ADDRESS should include the port, and then we can create DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT or something.  

Or alternately, we could try to make everything common and have *_ADDRESS_IP, *_ADDRESS_PORT, and then have users do "UPSTREAM_ADDRESS_IP:UPSTREAM_ADDRESS_PORT" in their format string; then they have have them together or separate.  But that raises more backwards-compat problems.  But whatever we come up with, it should be the most-consistent set of rules we can make.

Reply:
Yeah I had the same thought after I did this. Let me think about it and I will do another update.